docker/rumble/Dockerfile

```diff
@@ -29,5 +28,14 @@ RUN curl -k -L -O https://golang.org/dl/go1.16.4.linux-amd64.tar.gz && \
 ENV GOROOT=$WDIR/go
 ENV GOPATH=$WDIR/gopath
 ENV PATH $PATH:$GOROOT/bin:$WDIR
-# Set rumble jar location
+
+RUN git clone https://github.com/mrceyhun/rumble_jar.git && \
+    cp rumble_jar/spark-rumble-cmsmonit.jar $WDIR/spark-rumble.jar && \
+    rm -rf rumble_jar && \
+    git clone https://github.com/mrceyhun/CMSMonitoring.git && \
+    cp -R CMSMonitoring/src/go/Rumble/* $WDIR && \
+    rm -rf CMSMonitoring && \
+    go build rumble_server.go && \
```

Comment:
When building go executable always build static one, it can be done using `CGO_ENABLED=0 go build ...`. Doing this way you may build go executable in one image and then copy it to another without worrying about dependencies (and dynamic libs).

Reply:
I suggest to use additional flags to build static (compressed) executable, e.g.
```
CGO_ENABLED=0 go build -o rumble_server -ldflags="-s -w -extldflags -static" rumble_server.go
```